src/WorldClock.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import Clock from "./Clock";
+import { Container, Row, Col } from "react-bootstrap";
+
+class WorldClock extends React.Component {
+  render() {
+    const clockData = this.props.clockData;
+    return (
+      <Container>
+        {clockData.map((val, index) => (
```

Comment:
if we don't use the index, no need to define it here. Most likely your console will give you an error/warning however, in regards to not specifying a key on a collection. You would need to indicate the key property on the Row component here, and giving it the index as value seems alright most of the time